rules/sinks/third_parties/sdk/amplitude/java.yaml

```diff
@@ -9,5 +9,29 @@ sinks:
     domains:
       - "amplitude.com"
     patterns:
-      - "(?i)(com[.]amplitude|com[.]rudderstack[.]android[.]amplitude|com[.]rudderstack[.]android[.]integration[.]amplitude).*"
+      - "(?i)(com[.]rudderstack[.]android[.]amplitude|com[.]rudderstack[.]android[.]integration[.]amplitude).*"
```

Comment:
Update the `id`, `name` to include `rudderstack` in it

Reply:
updated